main/src/modules/bodySchema/sensoryProcessor/src/sensoryProcessor.cpp

```diff
@@ -282,10 +283,14 @@ bool SensoryProcessor::getMultimodalData()
     Size subPixWinSize(10,10), winSize(31,31);
 
     ImageOf<PixelRgb> *yarpImage = imgPortIn.read();
-    IplImage *cvImage = cvCreateImage(cvSize(yarpImage->width(), yarpImage->height()), IPL_DEPTH_8U, 3);
-    cvCvtColor((IplImage*)yarpImage->getIplImage(), cvImage, CV_RGB2BGR);
+    ImageOf<PixelBgr> tmp; tmp.resize(*yarpImage);
```

Comment:
`resize()` doesn't copy out the image, but only reshapes the image width and height equal to the input image.

the choice of `tmp` was only because it's a temporary object :smirk: